tock/employees/signals.py

```diff
@@ -0,0 +1,33 @@
+import logging
```

Comment:
I'm not saying there's anything wrong here, but I definitely like overriding `save()` on the model more than signals. I think it's more transparent, less likely to fail silently, and avoids weird circular dependency problems. Again, not asking for any changes, just throwing it out there.

Reply:
I'm +1 with on you @tbaxter-18f regarding the signals in general, but I have also run through this with @rogeruiz and we're good to go here!